test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -38,146 +38,230 @@ tags:
-   ```
+    ```shell script
+    open "https://$(oc get route alertmanager-route -n redhat-rhoam-middleware-monitoring-operator -o jsonpath='{.spec.host}')"
+    ```
 
-   Go to `Status -> Config` and check that the BU, SRE and Customer email addresses are included in the `BUAndCustomer` and `SRECustomerBU` receivers in the Alert Manager configuration where appropriate.
+    Go to `Status -> Config` and check that the BU, SRE and Customer email addresses are included in the `BUAndCustomer` and `SRECustomerBU` receivers in the Alert Manager configuration where appropriate.
 
-7. From the Openshift console, retrieve the 3Scale admin password by going to `Secrets` > `system-seed` under the `redhat-rhoam-3scale` namespace and copying the `admin` password
+7.  From the Openshift console, retrieve the 3Scale admin password by going to `Secrets` > `system-seed` under the `redhat-rhoam-3scale` namespace and copying the `admin` password
 
-8. Next, open the 3Scale admin console and login with `admin` as the username and the retrieved password
+8.  Next, open the 3Scale admin console and login with `admin` as the username and the retrieved password
 
-   ```sh
-   open "https://$(oc get route -n redhat-rhoam-3scale | grep 3scale-admin | awk {'print $2'})"
-   ```
+    ```shell script
+    open "https://$(oc get route -n redhat-rhoam-3scale | grep 3scale-admin | awk {'print $2'})"
+    ```
 
-9. Click on `Ok, how does 3scale work?` and follow the 3Scale wizard to create an API
+9.  Click on `Ok, how does 3scale work?` and follow the 3Scale wizard to create an API
 
 10. Once on your API overview page, click on `Integration` on the left, then on `Configuration`
 
 11. Take note of the `example curl for testing` for `Staging-APIcast`
 
-12. Trigger API Usage alerts
+12. Verify RHOAMApiUsageOverLimit alert is present
 
-    | Alert to fire                        | `numRequests` | `recipients`          |
-    | ------------------------------------ | ------------- | --------------------- |
-    | RHOAMApiUsageLevel1ThresholdExceeded | 43            | BU and Customers      |
-    | RHOAMApiUsageLevel2ThresholdExceeded | 46            | BU and Customers      |
-    | RHOAMApiUsageLevel3ThresholdExceeded | 51            | BU, Customers and SRE |
+    It takes 30 minutes for the RHOAMApiUsageOverLimit to fire. In a following step the number of requests required to trigger this
+    alert will be reached. You will be asked to note the time at that point. Ensure the alert is present by taking the following steps:
+
+    Navigate to the Prometheus Alert Console
+
+    ```shell script
+    open "https://$(oc get route prometheus-route -n redhat-rhoam-middleware-monitoring-operator -o jsonpath='{.spec.host}')/alerts"
+    ```
 
+    Verify that the alert query matches the below query:
+
+    `max_over_time((increase(ratelimit_service_rate_limit_apicast_ratelimit_generic_key_slowpath_total_hits[1m]))[30m:]) / 2 > 100`
```

Comment:
Do we need to specify the full query here? Or do we just want to validate that the updated rate limit config of `100` is reflected? If so we might just say that as the query could get updated and we'll need to update this test case